Makefile

```diff
@@ -776,15 +776,15 @@ $(PROGRAM_PREFIX)yosys-config: misc/yosys-config.in
 
 ABC_SOURCES := $(wildcard $(YOSYS_SRC)/abc/*)
 
-abc/abc$(EXE) abc/libabc.a: $(ABC_SOURCES)
+$(YOSYS_SRC)/abc/abc$(EXE) $(YOSYS_SRC)/abc/libabc.a: $(ABC_SOURCES)
```

Comment:
An out-of-tree build of Yosys should also do an out-of-tree build of abc.

Reply:
Should the abc executable be located in build in that case?

Reply:
Yep, along with all other files generated during the build. Running `git clean -dxf` in the Yosys or abc checkout should do nothing. (Modulo the odd case of abc being not actually a checked-out submodule.)